todolist/app.go

```diff
@@ -25,6 +25,10 @@ func (a *App) AddTodo(input string) {
 	a.Load()
 	parser := &Parser{}
 	todo := parser.ParseNewTodo(input)
+	if todo == nil {
+		fmt.Println("What to do?")
```

Comment:
Showing the whole manual in `usage()` is kind of overkilling, but for now we cannot show partial help messages just for the add option.  

While the message "what to do?" is not that informative, it does block the meaningless command ( often caused by typo ) quite well.